### DISCUSSION TITLE
Prisma Adapter not working with Email Provider

### DISCUSSION THREAD

**Discussion by Xaohs** [nextauthjs/next-auth#4501] [Help]
I am trying to make the email provider work with the prisma adapter. And am getting the following error upon confirming the email -> 
![image](https://user-images.githubusercontent.com/43537746/166441183-735fd1d6-f4d4-4919-98a6-6f38ba5f02c3.png).

The error is pretty straight forward, so I changed the "token" field to what it asks for "identifier_token", which then netted me a different error -> 
![image](https://user-images.githubusercontent.com/43537746/166441459-57a030bd-d28b-4b59-a719-37f750e1a4fe.png)

I am unsure of what I am missing in my database for it to not be working. 

This is my schema.
![image](https://user-images.githubusercontent.com/43537746/166441783-efe6b4e4-1329-4d9c-a2e7-1d520f1813d7.png)

Here is my [...nextauth]
![image](https://user-images.githubusercontent.com/43537746/166441902-7f0a1198-e04d-41b4-b414-7f44bab51973.png)

**Comment by blazephoenix** [comment]
Been having this myself too, the adapter is not configured for v4?

**Comment by SebastienPingal** [comment]
I have an issue also :
 code: 'MISSING_ADAPTER_METHODS_ERROR'

**Reply by Dinuda** [reply]
I have explained the solution at https://github.com/nextauthjs/next-auth/discussions/4059#discussioncomment-7743350

**Comment by Dinuda** [comment]
If anyone is facing the issue: `MISSING_ADAPTER_METHODS_ERROR`

I have explained the solution at https://github.com/nextauthjs/next-auth/discussions/4059#discussioncomment-7743350
